librz/core/cmd_debug.c

```diff
@@ -4265,6 +4265,56 @@ RZ_IPI RzCmdStatus rz_cmd_debug_step_until_flag_handler(RzCore *core, int argc,
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_cmd_debug_start_trace_session_handler(RzCore *core, int argc, const char **argv) {
+	if (rz_debug_is_dead(core->dbg)) {
+		eprintf("Cannot start session outside of debug mode, run ood?\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	if (core->dbg->session) {
```

Comment:
In the future we might have a simple API something like this:
```c
RZ_API bool rz_debug_is_session_active(RzDebug *dbg)
```
Like we have no `rz_debug_is_dead()`. It will be very helpful also for use from non-C bindings of the API.